oasis/deployments.md

```diff
@@ -166,6 +172,19 @@ and automatically accepted by most web browsers. The certificate used on port
 a self-signed certificate is recommended.
 {% endhint %}
 
+## How to enable the root user password rotation
+
+1. Navigate to the __Deployment__ for which you want to enable an automatic
+   password rotation for the root user.
+2. On the __Overview__ tab, click the button with the __cog__ icon next to the
+   __ROOT PASSWORD__.
+3. __Password Settings__ dialog, turn the password rotation on and click the
+   __Confirm__ button.
+
+   ![Oasis Deployment Password Rotation](images/oasis-deployment-password-rotation.png)
```

Comment:
We need to adjust the message displayed on the GUI:

Opt-in for password rotation for the root user --> Enable password rotation for the root user

Description --> When enabled, the root password will be automatically rotated every three months. You'll be notified about the rotation by email. When this option is turned off, the root password will not be rotated.

Reply:
@nikhil-varma: Can you take care of it? The source code appears to be here:

- https://github.com/arangodb-managed/dashboard/blob/master/frontend/src/dashboard/deployment/OverviewViews.tsx#L245
- https://github.com/arangodb-managed/dashboard/blob/4ead56e9ad25c819e51f23082eb7a31f3a3a1d97/frontend/src/dashboard/deployment/PasswordSettingsModalView.tsx#L93-L101

Reply:
@Simran-B we adjusted the texts with Nikhil, the GUI will be updated soon.